src/test/librbd/object_map/test_mock_ResizeRequest.cc

```diff
@@ -29,6 +29,18 @@ class TestMockObjectMapResizeRequest : public TestMockFixture {
                   exec(oid, _, StrEq("lock"), StrEq("assert_locked"), _, _, _))
                     .WillOnce(DoDefault());
     }
+    bool rwl_enabled = ictx->config.template get_val<bool>("rbd_rwl_enabled");
```

Comment:
Nit: same comment here (and throughout)

Reply:
Moved this part to expect_unlock_exclusive_lock() in src/test/librbd/test_mock_fixture.cc. As during open/close, it triggers the initialization of rwl, which calls set_feature, metadata_set/remove.